src/main/java/lotto/constant/ErrorText.java

```diff
@@ -0,0 +1,25 @@
+package lotto.constant;
+
+public enum ErrorText {
+    BLANK("빈칸은 입력할 수 없습니다."),
+    EMPTY_SPACE("공백을 포함하지 않는 숫자를 입력해주세요."),
+    NOT_INTEGER("숫자를 입력해주세요."),
+    OUT_OF_INTEGER_RANGE("정수 입력 범위를 초과했습니다."),
+    OUT_OF_MAX_AMOUNT("1,000원 ~ 100,000원 사이의 금액을 입력해주세요."),
+    THOUSAND_UNITS("구매금액은 1,000원 단위로 입력해주세요."),
+    DUPLICATE_NUMBER("당첨 번호와 중복 되지 않는 1~45사이의 수를 입력해 주세요."),
+    END_WITH_NUMBER("6개의 숫자를 입력해 주세요."),
+    INSUFFICIENT_LOTTO_NUMBER("당첨 번호 6개를 모두 입력해 주세요."),
+    NOT_LOTTO_NUMBER("로또 번호는 1~45사이의 숫자만 가능 합니다."),
+    DUPLICATE_WINNING_NUMBER("당첨 번호는 중복 되지 않아야 합니다.");
+
+    private String message;
+
+    ErrorText(String message) {
+        this.message = message;
+    }
+
+    public String getMessage() {
+        return message;
```

Comment:
개인적으로는 에러 메시지 앞에 붙는 "[ERROR] " 역시 에러 메시지와 같은 관심사라고 생각해요.
해당 메시지도 내부 상수로 같이 관리하는 건 어땠을까요?
```suggestion
        private final static PREFIX = "[ERROR] "
        return PREFIX + message;
```

Reply:
zangsu님 말씀을 듣고 생각해보니 태그도 이곳에 있는 게 맞는 것 같아요
사실 ErrorText에서는 메세지 문자열만 관리하고 싶어서 '[ERROR]'태그를 다른 enum으로 뺐었어요
enum안에서 상수를 또 선언할 생각을 미처 못했네요 그래서 지금처럼 분리가 되어버린 것 같아요
이런 방법도 있었군요.. 클래스에서는 잘 썼는데 상수를 다시 enum으로 바꾸는 과정에서 놓친 것 같습니다ㅠㅠ
덕분에 하나 더 배워갑니다!ㅎㅎ